client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -439,6 +441,7 @@ export class PlanFeatures2023Grid extends Component< PlanFeatures2023GridType >
 			product_slug: storageAddOn.productSlug,
 			quantity: storageAddOn.quantity,
 			volume: 1,
+			extra: { feature_slug: selectedStorageOption },
```

Comment:
**Note:**
Because the add on `product_slug` is identical for both 50GB and 100GB storage add-ons, we add `feature_slug` data to Add On Cart Item extra parameters. That way we can dispatch a tracks event with a human readable slug in https://github.com/Automattic/wp-calypso/pull/81816/files#diff-18214fd84316736ff6662f03fa201b6255f4c20f277c0eb683b5e36e9c9dea6dR300-R317

Reply:
`extra` has become kind of a dumping ground for all sorts of metadata that the client needs to pass to the server when adding a product, so I think this is probably fine.